$NetBSD: patch-pyvex__c_Makefile,v 1.2 2020/01/09 14:02:32 wiz Exp $

Add support for NetBSD.
Correct the syntax for GCC in C99 mode.

--- pyvex_c/Makefile.orig	2019-12-17 03:13:10.000000000 +0000
+++ pyvex_c/Makefile
@@ -2,22 +2,27 @@ UNAME := $(shell uname)
 ifeq ($(UNAME), Darwin)
 	LIBRARY_FILE=libpyvex.dylib
 	STATIC_LIBRARY_FILE=libpyvex.a
-	LDFLAGS=-Wl,-install_name,$(LIBRARY_FILE)
+	LDFLAGS+=-Wl,-install_name,$(LIBRARY_FILE)
 endif
 ifeq ($(UNAME), Linux)
 	LIBRARY_FILE=libpyvex.so
 	STATIC_LIBRARY_FILE=libpyvex.a
-	LDFLAGS=-Wl,-soname,$(LIBRARY_FILE)
+	LDFLAGS+=-Wl,-soname,$(LIBRARY_FILE)
 endif
 ifeq ($(UNAME), NetBSD)
 	LIBRARY_FILE=libpyvex.so
 	STATIC_LIBRARY_FILE=libpyvex.a
-	LDFLAGS=-Wl,-soname,$(LIBRARY_FILE)
+	LDFLAGS+=-Wl,-soname,$(LIBRARY_FILE)
+endif
+ifeq ($(UNAME), NetBSD)
+	LIBRARY_FILE=libpyvex.so
+	STATIC_LIBRARY_FILE=libpyvex.a
+	LDFLAGS+=-Wl,-soname,$(LIBRARY_FILE)
 endif
 ifeq ($(findstring MINGW,$(UNAME)), MINGW)
 	LIBRARY_FILE=pyvex.dll
 	STATIC_LIBRARY_FILE=libpyvex.a
-	LDFLAGS=
+	LDFLAGS+=
 endif
 
 # deeply evil
